region/maridia/outer.json

```diff
@@ -3306,6 +3620,34 @@
                 }                    
               ]
             },
+            {
+              "id": 6,
+              "strats": [
+                {
+                  "name": "G-Mode Morph",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Gravity",
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      {"and": [
+                        "h_canArtificialMorphSpringBallBombJump",
```

Comment:
Springball+hijump
springball+gravity jump
bomb+gravity jump

Reply:
bomb + gravity jump is really interesting, but I'm going to leave it off, since there's IBJ